### ISSUE TITLE
Provide images!

### ISSUE THREAD

**Issue by dennisivy** [garzo94/Video-Meting-Agora-app-Frontend#1]
Please add some images in a folder or to the readme file.

**Comment by dennisivy** [comment]
Oops, nvm, you already did this ;)

**Comment by garzo94** [comment]
Yes, there's a folder with the images, but I don't know what happened, I was sure I added a readme file.
this is the repo backend: https://github.com/garzo94/Video-Meeting-Agora-app-Backend...
And this the Figma link https://www.figma.com/file/OWblJFRn0Dn02Y6rOmuBYC/Rapid-Meeting?node-id=0%3A1
